init.lua

```diff
@@ -108,6 +109,16 @@ function lint.get_linter_for_doc(doc)
     if common.match_pattern(file, linter.filename) then
       return linter, name
     end
+    if linter.syntax ~= nil then
+      local header = doc:get_text(1, 1, doc:position_offset(1, 1, 128))
+      local syn = syntax.get(doc.filename or "", header)
+      for i = #linter.syntax, 1, -1 do
+         local s = linter.syntax[i]
```

Comment:
I think this block is mistakenly indented with 3 spaces instead of 2.

Reply:
Thanks for the notes, changes have been pushed